src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -47,6 +48,7 @@ export class RiddleOfFire extends BuffWindow {
 	@dependency globalCooldown!: GlobalCooldown
 
 	private blitzActions = fillActions(BLITZ_ACTIONS, this.data)
+	private riddleActions = fillActions(['RIDDLE_OF_WIND'], this.data)
```

Comment:
If it's a single action and will only ever be a single action, why does this need to fill actions?

Reply:
No reason, I'll get rid of it